Makefile

```diff
@@ -0,0 +1,32 @@
+JEKYLL_VERSION ?= 4.2.2
+
+CONTAINER_NAME ?= flotta.github.io
```

Comment:
since this isn't flotta's specific, a genetic name related to this project would be a better fit for this project.

Reply:
Good point